modules/social_features/social_featured_content/templates/node--event--featured.html.twig

```diff
@@ -4,13 +4,15 @@
     <small class="text-muted">
       {% embed "node--teaser__field.html.twig" %}
         {%- block field_icon -%} event {%- endblock -%}
+        {% block svg_title %} <title>{% trans %} Event date: {% endtrans %}</title> {% endblock %}
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
All SVG icons are displayed via `node--teaser__field.html.twig` template have a `aria-hidden='true'` attribute